src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/impl/SolutionGraph2SolutionMappingImpl.java

```diff
@@ -0,0 +1,35 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QueryExecutionFactory;
+import org.apache.jena.query.ResultSet;
+import org.apache.jena.rdf.model.Model;
+
+import se.liu.ida.hefquin.engine.data.SolutionMapping;
+import se.liu.ida.hefquin.engine.data.impl.SolutionMappingImpl;
+import se.liu.ida.hefquin.engine.query.SPARQLQuery;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.SolutionGraph2SolutionMappings;
+
+public class SolutionGraph2SolutionMappingImpl implements SolutionGraph2SolutionMappings {
+
+    @Override
+    public List<SolutionMapping> querySolutionGraph(final Model solutionGraph, 
+                                                    final SPARQLQuery query) throws RuntimeException {
```

Comment:
I don't think that Runtime exceptions need to be declared in the method signature.